extensions/data-plane/data-plane-control-api/README.md

```diff
@@ -0,0 +1,51 @@
+# Data Plane API
+
+This extension provides the APIs of the Data Plane which are used by the Control Plane to delegate the data transfer 
+to its Data Plane. APIs are not intended for public use. [See Data Plane Public API](../data-plane-public-api/README.md).
+
+### Scope
+
+These APIs have been designed to support the different data transfer types that the Data Plane must support, which are:
+
+- Consumer Data Pull: the consumer backend applications use the Data Plane to proxy data request until the provided data source.
+  This data transfer type is exclusively built around the [Data Plane Public API](../data-plane-public-api/README.md).
```

Comment:
the control API are used only for the Provider Push mechanism, the public for the Consumer Pull